docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -70,6 +70,67 @@ This test verifies whether the battery charge level is reported in the OS.
 1. The battery status indicator should show the current charge level (percentage
     left).
 
+## ECR001.003 Battery start charge threshold
+
+**Test description**
+
+The setup menu allows you to set your own battery thresholds, setting the
+minimum and maximum battery charge levels. The purpose of this test is to check
```

Comment:
As above, this test is not correct for the implemented functionality.

Reply:
fixed in 0aded10aff1c3a205123dc518c700ca3beee752a

Reply:
Thanks